lightning/src/ln/channelmanager.rs

```diff
@@ -1122,7 +1122,8 @@ pub enum RecentPaymentDetails {
 	/// been resolved. Upon receiving [`Event::PaymentSent`], we delay for a few minutes before the
 	/// payment is removed from tracking.
 	Fulfilled {
-		/// Hash of the payment that was claimed.
+		/// Hash of the payment that was claimed. Returns `None` for possibly older serializations
```

Comment:
Let's be specific. Looks like the optional payment hash was introduced in 0.0.104 (https://github.com/lightningdevkit/rust-lightning/commit/2c4f16d5e39e6d4e90d1b9836cceaad11810fbaa).

Reply:
resolved in 7ce4291e